task4/src/main/scala/com/bootcamp/task/Homework.scala

```diff
@@ -0,0 +1,129 @@
+package com.bootcamp.task
+
+import scala.annotation.tailrec
+
+object Homework {
+
+	// try to implement min different ways (fold, reduce, recursion)
+	def min(list: List[Int]): Option[Int] = {
```

Comment:
8 spaces is a lot and works bad with Scala style of coding. We usually use just 2

Reply:
8 spaces? It's a tab char here...you can configure it easily to display as 2,4,8 spaces, that's the advantage of a tab vs spaces =D

Reply:
o i see im sorry, im not used to it  :)

Reply:
Got it, team uses spaces in projects?

Reply:
Yes, its spaces in projects i work on